Problem_1.cpp

```diff
@@ -0,0 +1,29 @@
+/*
+ * Time Complexity : O(n*amount)
+ * Space Complexity : O(1)
```

Comment:
Check space complexity

Reply:
Yes, space complexity would be O(amount)